### ISSUE TITLE
Flawed select features buttons

### ISSUE THREAD

**Issue by fzadrazil** [hslayers/hslayers-ng#2147]
# Bug

**Describe the bug**
When clicking on the Selection button in the draw toolbar, layer dropdown opens too.
Selection button in the draw panel does not work at all. It seems to not switch the active tool.

**To Reproduce**
1. Go to https://hub.lesprojekt.cz/map3/
2. Click on Selection button in the draw toolbar

**Screenshots**
![image](https://user-images.githubusercontent.com/48727502/132917967-b892eccb-82b7-424d-a6d9-9135c4b85ed6.png)
![image](https://user-images.githubusercontent.com/48727502/132918401-30f10f64-6d0c-42af-9b94-e80dd87eb682.png)

**Additional context**
It seems to occur only in hslayers-app. It works fine in the full app.

**Comment by FilipLeitner** [comment]
1) Selection button in the draw panel does not work at all. It seems to not switch the active tool.
- I actually did that on purpose as the 'Selectiom' button was changed to dropdown but I get the point that its counter-intuitive to have button behave differently.

2) When clicking on the Selection button in the draw toolbar, layer dropdown opens too.
- wasnt able to reproduce localy on either hslayers-app (production), test-app (developement), full example (dist linked)
